fog/distribution/src/config.rs

```diff
@@ -99,7 +99,7 @@ impl Config {
     ) -> ConnectionResult<Vec<SyncConnection<ThickClient<HardcodedCredentialsProvider>>>> {
         let mut mr_signer_verifier =
             MrSignerVerifier::from(mc_consensus_enclave_measurement::sigstruct());
-        mr_signer_verifier.allow_hardening_advisory("INTEL-SA-00334");
+        mr_signer_verifier.allow_hardening_advisories(&["INTEL-SA-00334", "INTEL-SA-00615"]);
```

Comment:
Might as well use the measurement constants (across this PR).

```suggestion
        mr_signer_verifier.allow_hardening_advisories(mc_consensus_enclave_measurement::HARDENING_ADVISORIES);
```